contracts/helpers.cairo

```diff
@@ -314,3 +327,49 @@ func fromUint256_balance{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_c
     }
     return x__;
 }
+
+
+// equivalent to toUint256_balance, but for option_position, which is stored as a felt (Int),
+// because we don't need the full range of uint256
+func toInt_balance{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    x: Math64x61_,
+    currency_address: Address
+) -> Int {
+    let (decimal) = get_decimal(currency_address);
+    let (dec_) = pow10(decimal);
+    let x_ = x * dec_;
+
+    with_attr error_message("x_ out of bounds in toUint256_balance"){
+        assert_le(x, Math64x61.BOUND);
+        assert_le(-Math64x61.BOUND, x);
```

Comment:
then I guess `assert_le(-Math64x61.BOUND, x);` is completely redundant...

Reply:
No, m64x61 value could be negative and then toFelt won't fail since felts are signed, but uint256 are not signed so it would fail silently. Changed to `assert_le(0, x);`